lib/blockchain_double.js

```diff
@@ -546,6 +547,11 @@ BlockchainDouble.prototype.processNextBlock = function(callback) {
   this.createBlock(function(err, block) {
     if (err) return callback(err);
 
+    //Overwrite block timestamp
+    if (timestamp) {
+        block.header.timestamp = to.hex(timestamp);
```

Comment:
Is it a requirement that block timestamps be monotonic? If so, we should add a check for this here.

Reply:
Even if it _isn't_ a requirement that block timestamps be monotonic, I think we should at least consider logging a warning message or something to that effect when `timestamp` is earlier than the parent block's timestamp.

Reply:
It's a good idea, I've added the warning.